package.json

```diff
@@ -22,50 +20,64 @@
     "type": "git",
     "url": "git://github.com/opentypejs/opentype.js.git"
   },
-  "main": "dist/opentype.js",
-  "module": "dist/opentype.module.js",
-  "bin": {
-    "ot": "./bin/ot"
-  },
+  "main": "./dist/opentype.js",
+  "browser": "./dist/opentype.js",
+  "module": "./dist/opentype.module.js",
   "scripts": {
-    "start": "node ./bin/server.js",
-    "watch": "rollup -c -w",
-    "test": "mocha --require reify --recursive && jshint . && jscs .",
-    "build": "rollup -c",
-    "minify": "uglifyjs --source-map \"url='opentype.min.js.map'\" --compress --mangle --output ./dist/opentype.min.js -- ./dist/opentype.js",
-    "dist": "npm run test && npm run build && npm run minify"
-  },
-  "files": [
-    "LICENSE",
-    "RELEASES.md",
-    "README.md",
-    "bin",
-    "dist",
-    "src"
-  ],
-  "engines": {
-    "node": ">= 8.0.0"
+    "build": "npm run b:umd && npm run b:esm",
+    "dist": " npm run d:umd && npm run d:esm",
+    "test": "npm run build && npm run dist && mocha --require reify --recursive",
```

Comment:
Didn't we want to add lining after the testing?

Reply:
you want `npm run test` to do the linting ?

Reply:
Right now we run linting and coding style at the end of the test script

Reply:
I would have prefered to keep them separated (CI will check both anyway) but if that's how it was before... I'll add it back

Reply:
I'd like running them before I commit or push anything, and knowing myself I'll surely forget the linting :)

Reply:
I'll leave the `lint` target since it's a burden to re-run all unit test only to get the linting error

Reply:
Absolutely! And we don't need to run linting twice during CI, so it just needs to call the test script

Reply:
Done :heavy_check_mark:

Reply:
The `version` is set at publication 
see https://github.com/yne/opentype.yne.js/blob/master/.github/workflows/release.yml#L22

Reply:
the `version` is set by github action upon git tag
See https://github.com/yne/opentype.yne.js/blob/master/.github/workflows/release.yml#L22

the `name` is also automatically so anybody can fork and rename they fork so they can publish they own fork without conflicting when PR